apps/mobile/src/components/GalleryEditor/GalleryEditorActiveActions.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback } from 'react';
+import { GestureResponderEvent, View } from 'react-native';
+import { graphql, useFragment } from 'react-relay';
+import { DragIcon } from 'src/icons/DragIcon';
+import { MinusCircleIcon } from 'src/icons/MinusCircleIcon';
+import { PlusCircleIcon } from 'src/icons/PlusCircleIcon';
+
+import { useGalleryEditorActions } from '~/contexts/GalleryEditor/GalleryEditorContext';
+import { StagedRow } from '~/contexts/GalleryEditor/types';
+import { GalleryEditorActiveActionsFragment$key } from '~/generated/GalleryEditorActiveActionsFragment.graphql';
+import useMaxColumnsGalleryEditor from '~/shared/hooks/useMaxColumnsGalleryEditor';
+
+import { GalleryTouchableOpacity } from '../GalleryTouchableOpacity';
+import { BaseM } from '../Text';
+
+type Props = {
+  row: StagedRow;
+  queryRef: GalleryEditorActiveActionsFragment$key;
+};
+
+export function GalleryEditorActiveActions({ row, queryRef }: Props) {
+  const query = useFragment(
+    graphql`
+      fragment GalleryEditorActiveActionsFragment on Query {
+        viewer {
+          __typename
+          ...useMaxColumnsGalleryEditorFragment
+        }
+      }
+    `,
+    queryRef
+  );
+
+  if (query.viewer?.__typename !== 'Viewer') {
+    throw new Error('Expected viewer to be present');
+  }
+
+  const { incrementColumns, decrementColumns } = useGalleryEditorActions();
+
+  const maxColumns = useMaxColumnsGalleryEditor(query.viewer);
```

Comment:
non-blocking: we can stop using this and hard-code max_columns to be `6` on both web and mobile! can refactor later

Reply:
Sounds good! I can refactor this later